lib/bes/buildbuddy.go

```diff
@@ -0,0 +1,140 @@
+package bes
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/cookiejar"
+	"net/url"
+
+	"github.com/enfabrica/enkit/lib/client"
+	bespb "github.com/enfabrica/enkit/third_party/bazel/buildeventstream"
+	bbpb "github.com/enfabrica/enkit/third_party/buildbuddy/proto"
+
+	"github.com/golang/protobuf/proto"
+)
+
+var (
+	BuildBuddyProdURL    = mustParseURL("https://buddy.bazel.corp.enfabrica.net")
+	BuildBuddyStagingURL = mustParseURL("https://buddy.staging-bazel.corp.enfabrica.net")
```

Comment:
Should we check these into enkit, make the caller always provide them, or something else?